### PR TITLE
[IMP] l10n_ar: suggest search by company in account invoice report

### PR DESCRIPTION
**Pull request by bruno-zanotti**
**task 426**

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by ren-odoo** [comment]
Hi @jco-odoo 

Can you help me to take a look at this small improvement.

File: addons/l10n_ar/report/invoice_report_view.xml [hunk 1]
```diff
@@ -7,6 +7,7 @@
         <field name="inherit_id" ref="account.view_account_invoice_report_search" />
         <field name="arch" type="xml">
             <search>
+                <field name="company_id"/>
```
Comment (jco-odoo) [inline addons/l10n_ar/report/invoice_report_view.xml:10]: @Bruno-Zanotti groups="base.group_multi_company"  I don't know if it makes that much sense.  A group by company seems more logical.  @ren-odoo

**Comment by ren-odoo** [comment]
@Bruno-Zanotti add the company group by as part of the default groups and we can remove the filter.

**Comment by ren-odoo** [comment]
@jco-odoo This PR is ready for you last review and merge. 


Thank you.

File: addons/l10n_ar/report/invoice_report_view.xml [hunk 2]
```diff
@@ -10,7 +10,7 @@
                 <field name="l10n_ar_state_id"/>
                 <filter name="with_document" string="With Document" domain="[('l10n_latam_document_type_id', '!=', False)]"/>
             </search>
-            <filter name="user" position="after">
+            <filter name="company" position="after">
```
Comment (jco-odoo) [inline addons/l10n_ar/report/invoice_report_view.xml:13]: Why this change?

File: addons/l10n_ar/report/invoice_report_view.xml [hunk 3]
```diff
@@ -21,7 +21,7 @@
         <field name="name">IIBB - Sales by jurisdiction</field>
         <field name="res_model">account.invoice.report</field>
         <field name="view_mode">pivot</field>
-        <field name="context">{'search_default_current': 1, 'search_default_customer': 1, 'search_default_with_document': 1, 'search_default_groupby_l10n_ar_state_id': 1, 'search_default_groupby_account_id': 1, 'time_ranges': {'field': 'date', 'range': 'this_year'}}</field>
+        <field name="context">{'search_default_current': 1, 'search_default_customer': 1, 'search_default_with_document': 1, 'search_default_company': 1, 'search_default_groupby_l10n_ar_state_id': 1, 'search_default_groupby_account_id': 1, 'time_ranges': {'field': 'date', 'range': 'this_year'}}</field>
```
Comment (jco-odoo) [inline addons/l10n_ar/report/invoice_report_view.xml:24]: I suppose this is the group by company and we should maybe have given it a better name.

**Review by jco-odoo — COMMENTED** [review]
Please check

File: addons/l10n_ar/report/invoice_report_view.xml (hunk 2 above)
Comment (Bruno-Zanotti) [reply addons/l10n_ar/report/invoice_report_view.xml]: That was to put the group by company before the group by state and account, but actually it wasn't necessary. We achieve this with the context priorities.

File: addons/l10n_ar/report/invoice_report_view.xml (hunk 3 above)
Comment (Bruno-Zanotti) [reply addons/l10n_ar/report/invoice_report_view.xml]: @jco-odoo Yes, that's the group by company field from [here](https://github.com/odoo/odoo/blob/65b0338883e173c792ee177eb438d411f214bb8d/addons/account/report/account_invoice_report_view.xml#L90). I do not know if we can, but if you prefer we can change it

**Comment by vbe-odoo** [comment]
Hi @Bruno-Zanotti 
Hope you are doing well!

I've done the test and all looks good to go!
I am moving this task to Technical Testing and assigning it to JOC
Dummy PR used: https://github.com/odoo/enterprise/pull/15661
Comment left on the main PR: https://github.com/odoo/odoo/pull/62702

Thanks!
Bests
--

**Comment by jco-odoo** [comment]
@robodoo r+
